Cargo.toml

```diff
@@ -13,29 +13,22 @@ exclude = ["cmd/crates/soroban-test/tests/fixtures/hello"]
 [workspace.dependencies.soroban-env-host]
 version = "0.0.15"
 git = "https://github.com/stellar/rs-soroban-env"
-rev = "5ef87058f07ad67eeba4c350a6b6af82bd17f916"
+rev = "83ab1b0e2ce7e08d33d686ed4b4ea0564b454160"
 
 [workspace.dependencies.soroban-spec]
 version = "0.7.0"
 git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "29b505cf95f2fd4fbe639fdb163b606c44920837"
-
-
-[workspace.dependencies.soroban-token-spec]
```

Comment:
would it be possible to update it with the new hash ? `9bbac60c25cee9aa995025db5af828c243d15690` ?

Reply:
That's the version I updated everything to

Reply:
soroban-token-spec was deleted upstream, if that's what you are referring to

Reply:
oh - I did not realized that it was deleted upstream.